<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UIDemo01</class>
 <widget class="QDialog" name="UIDemo01">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetTitle" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labIco">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labTitle">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widgetTop" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="btnMain">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTipDuration">
            <number>-5</number>
           </property>
           <property name="text">
            <string>开始采集</string>
           </property>
           <property name="icon">
            <iconset resource="image/jjj.qrc">
             <normaloff>:/new/start_102.82503192848px_1231086_easyicon.net.png</normaloff>:/new/start_102.82503192848px_1231086_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnConfig">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>参数设置</string>
           </property>
           <property name="icon">
            <iconset resource="image/jjj.qrc">
             <normaloff>:/new/settings_115.24904214559px_1231079_easyicon.net.png</normaloff>:/new/settings_115.24904214559px_1231079_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnData">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>缩放模式</string>
           </property>
           <property name="icon">
            <iconset resource="image/jjj.qrc">
             <normaloff>:/new/View_Width_128px_1178984_easyicon.net.png</normaloff>:/new/View_Width_128px_1178984_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnHelp">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>数据保存</string>
           </property>
           <property name="icon">
            <iconset resource="image/jjj.qrc">
             <normaloff>:/new/params_128px_1158068_easyicon.net.png</normaloff>:/new/params_128px_1158068_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnExit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>用户退出</string>
           </property>
           <property name="icon">
            <iconset resource="image/jjj.qrc">
             <normaloff>:/new/access_close_exit_logout_sign_out_128px_1225473_easyicon.net.png</normaloff>:/new/access_close_exit_logout_sign_out_128px_1225473_easyicon.net.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widgetMenu" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,5">
      <item>
       <widget class="QWidget" name="PlotArea" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Channel</string>
           </property>
           <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0,0,0">
            <item row="8" column="1">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>reset</string>
              </property>
             </widget>
            </item>
            <item row="5" column="4">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="2">
             <widget class="QComboBox" name="comboBox_2">
              <item>
               <property name="text">
                <string>男</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>女</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="7" column="1">
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>姓名：</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="comboBox_3">
              <item>
               <property name="text">
                <string>HbO</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HbR</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>HbT</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ALL</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QComboBox" name="comboBox"/>
            </item>
            <item row="5" column="3">
             <widget class="QComboBox" name="comboBox_4">
              <item>
               <property name="text">
                <string>EEG1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>EEG2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ALL</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="5" column="2">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>性别：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>年龄：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="5" column="0">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="9" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="6" column="4">
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="6" column="0">
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item row="6" column="1">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>scale</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Analyze</string>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="7" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Scale:</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLineEdit" name="time_input"/>
            </item>
            <item row="8" column="1">
             <widget class="QComboBox" name="scale">
              <item>
               <property name="text">
                <string>10mV</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1mV</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>100uV</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10uV</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QComboBox" name="comboBox_5">
              <item>
               <property name="text">
                <string>fft</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>welch</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="11" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>OK</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pageButton2">
              <property name="text">
               <string>page2</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pageButton3">
              <property name="text">
               <string>page3</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="pageButton4">
              <property name="text">
               <string>page4</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pageButton1">
              <property name="text">
               <string>page1</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="hi" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Display</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QHBoxLayout" name="plotA"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image/jjj.qrc"/>
 </resources>
 <connections/>
</ui>
